### DISCUSSION TITLE
v2/zh/guide/feature/encrypt

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#62] [Announcements]
# v2/zh/guide/feature/encrypt

一个具有强大功能的 vuepress 主题✨

https://vuepress-theme-hope.github.io/v2/zh/guide/feature/encrypt.html

**Comment by Ink-jqs** [comment]
试了一下全局加密好像没效果，局部加密是可以的。不知道是为啥🤔

**Reply by Mister-Hope** [reply]
这个问题已经修复了
